docs/content/key-topics/_index.adoc

```diff
@@ -4,4 +4,61 @@ title: "Key Topics"
 weight: 2
 ---
 
-Coming soon...
+The following is a list of all satellite applications that were developed to facilitate the mission and a description of their basic functionality: 
```

Comment:
A reference in the form of a [footnote](https://docs.asciidoctor.org/asciidoc/latest/macros/footnote/#_footnotedef_1) may be good to append here